shared/spotify-api/src/mockClient.ts

```diff
@@ -163,56 +163,72 @@ export class MockSpotifyClient implements SpotifyDataSource {
     id: string,
     params?: { limit?: number; offset?: number; include_groups?: string }
   ): Promise<Spotify.Object.Paginated<Spotify.Object.AlbumSimplified>> {
-    throw new Error('Mock not implemented.');
+    const albums = Object.entries(mocks.albums).filter(
+      ([id, album]) => id == id
+    );
+    return {
+      total: albums.length,
+      limit: params?.limit ?? 10,
+      offset: params?.offset ?? 0,
+      href: 'http://mocked.com/artistAlbums',
+      next: 'http://mocked.com/artistAlbums',
+      previous: 'http://mocked.com/artistAlbums',
+      items: albums.values() as any,
+    };
   }
 
   async getArtistRelatedArtists(
     artistId: string
   ): Promise<Spotify.Object.List<'artists', Spotify.Object.Artist>> {
-    throw new Error('Mock not implemented.');
+    return { artists: [] };
   }
 
   async getArtistTopTracks(
     artistId: string,
     params: { market: string }
   ): Promise<Spotify.Object.ArtistTopTracks> {
-    throw new Error('Mock not implemented.');
+    const artist = mocks.artists[artistId];
+    if (!artist) return { tracks: [] };
+    const tracks = Object.entries(mocks.tracks)
+      .filter(([id, track]) => track.artists[0].id == artistId)
```

Comment:
```suggestion
      .filter(([id, track]) => track.artists.some((artist) => artist.id === artistId))
```

If you want to be a bit more thorough, you could use a `.some` here to check if any of the track's artists match the artist id.